lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -53,86 +78,142 @@ class ArgumentsForwarding < Base
 
         minimum_target_ruby_version 2.7
 
-        MSG = 'Use arguments forwarding.'
-
-        # @!method use_rest_arguments?(node)
-        def_node_matcher :use_rest_arguments?, <<~PATTERN
-          (args ({restarg kwrestarg} $_) $...)
-        PATTERN
-
-        # @!method only_rest_arguments?(node, name)
-        def_node_matcher :only_rest_arguments?, <<~PATTERN
-          {
-            (send _ _ (splat (lvar %1)))
-            (send _ _ (hash (kwsplat (lvar %1))))
-          }
-        PATTERN
-
-        # @!method forwarding_method_arguments?(node, rest_name, block_name, kwargs_name)
-        def_node_matcher :forwarding_method_arguments?, <<~PATTERN
-          {
-            (send _ _
-              (splat (lvar %1))
-              (block-pass {(lvar %2) nil?}))
-            (send _ _
-              (splat (lvar %1))
-              (hash (kwsplat (lvar %3)))
-              (block-pass {(lvar %2) nil?}))
-          }
-        PATTERN
+        FORWARDING_LVAR_TYPES = %i[splat kwsplat block_pass].freeze
+
+        FORWARDING_MSG = 'Use arguments forwarding.'
```

Comment:
Might be good to mention the specific syntax for this in the message, as I'm not sure everyone's familiar with those names.

Reply:
I somewhat agree, but this was the same message as previously used in this cop, so I think I'd favour leaving it as it is, unless you feel strongly about it?

Reply:
Let's change this, I just didn't notice it when the cop was originally introduced. You can do it in a separate commit or the same. It's all the same to me.